src/components/screens/wallet/overview/index.js

```diff
@@ -3,32 +3,23 @@ import { useSelector } from 'react-redux';
 import { compose } from 'redux';
 import { withTranslation } from 'react-i18next';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
-import { isEmpty } from '@utils/helpers';
 import withData from '@utils/withData';
 import { getTransactions } from '@utils/api/transaction';
 import BalanceChart from './balanceChart';
 import AccountInfo from './accountInfo';
 import BalanceInfo from './balanceInfo';
 import styles from './overview.css';
 
-const getProp = (dic, prop, defaultValue) => {
-  if (!dic || isEmpty(dic)) {
-    return defaultValue;
-  }
-  return dic[prop];
-};
-
 const Overview = ({
   t, activeToken, transactions, hwInfo,
   discreetMode, isWalletRoute, account,
 }) => {
-  const address = getProp(account, 'address', '');
-  const publicKey = getProp(account, 'publicKey', '');
-  const balance = getProp(account, 'balance', 0);
+  const { address, publicKey, balance } = account.info.LSK.summary;
```

Comment:
Do we need a null check here?

Reply:
Yes, and it should be `account.summary?`